statbot/sql.py

```diff
@@ -274,7 +307,7 @@ def _voice_channel_values(channel):
             'is_deleted': False,
             'position': channel.position,
             'bitrate': channel.bitrate,
-            'user_limit': channel.user_limit,
+            'user_limit': channel.user_limit or 0,
```

Comment:
why is the `bitrate` nullable but the `user_limit` 0?

Reply:
Discord represents a voice channel that can hold an unlimited number of users with the value of `0`.

`bitrate` is sort of weird. The API says [it should never be `None`](https://discordpy.readthedocs.io/en/rewrite/api.html#discord.VoiceChannel.bitrate), but while running statbot on the actual server, the value returned for one the VCs was `None`.

Reply:
but if discord represents it that way, why do you need to turn (what I assume is) a `None` to a `0`?

Reply:
I... uh, I'm not sure. After I posted that last comment, I thought maybe it can't get the data because of a permissions thing or something. I will investigate this.

Reply:
Alright I fixed this. It seems to be that the old version of discord.py was misinterpreting channel categories as voice channels for some reason.